### ISSUE TITLE
Query About ResponsiveMultiLevelMenu

### ISSUE THREAD

**Issue by Debuggingyourcode** [codrops/ResponsiveMultiLevelMenu#26]
Hi,

What can i use this ResponsiveMultiLevelMenu JQuery for free Wordress Plugin ?

**Comment by rejas** [comment]
Sorry, I dont understand your question, please rephrase it.

**Comment by Debuggingyourcode** [comment]
my query is what can is use your  ResponsiveMultiLevelMenu for My Wordpress plugin.
 your  ResponsiveMultiLevelMenu  is free ? 
 can i use in my project , wodrpress plugin etc... ?

**Comment by rejas** [comment]
Well, I am not a lawyer, but as long as you respect the MIT-License (which is the license of this plugin as I see it on the frontpage) it should be okay for you to use it. As for the license, here is a link to it: http://opensource.org/licenses/MIT

**Comment by native-apps** [comment]
I've integrated this into a WP theme. I've added many features, but one feature im hoping to get help with is having the trigger button outside of the nav.

**Comment by rejas** [comment]
whats your problem with that particularly? any demo code to see?

**Comment by native-apps** [comment]
YES! I guess, the main thing I really need RMLM to do is be able to have the trigger outside of the nav. this would fix a ton of layout issues i'm having. You can see it in action here. I'm happy to post the code if needed, as its broken up into my WP theme. I had to use the walker nav wp functions to get WP to structure the nav elements properly  and it works great. 

Here is a site using the RMLM now. 
https://www.movieboxtv.com/product/android-movie-box/

Since the trigger button has to be placed inside the nav menu to activate the function, I get a width bug during the menu animations, but I don't have that problem when viewed in mobile since everything is set to absolute positioning and 100% width...  Now, I would use absolute positioning in desktop mode but I don't want that since I have custom menu labels options in my theme, so the width needs to adjust accordingly. In mobile view those labels hide, but I do have to use absolute positioning in mobile view which is not ideal, but for now is working even though I have some z-index issues to overcome. So, being able to have the button trigger outside of the nav element would solve all of these problems, but I haven't edited the JS since i'm not a good js programmer.  Any help would be greatly appreciated as I would give my code out for free. I believe this menu is the best in the world.

**Comment by native-apps** [comment]
You can see the width bug during animation when the menu has sub menu items, not on initial animation of top level menus.

**Comment by rejas** [comment]
Take a look at branch "button-refactor" of my repo-fork at https://github.com/rejas/ResponsiveMultiLevelMenu/tree/button_refactor, there I moved the button away from the nav, changed the code to make it work again. Is that what you need / had in mind?

**Comment by native-apps** [comment]
Awesome!!! Thank you so much! I will test it later today and post my results as well as my wp code.

**Comment by native-apps** [comment]
Hmm. It works, but the problem is its now opening all menus or the wrong ones at the same time..

I added this code from an earlier post to enable multiple menus: https://github.com/codrops/ResponsiveMultiLevelMenu/issues/7

Then I added your new code, but It seems that I have no way to assign which button controls which menu. 
Also, is it necessary to remove the function to click on the body to hide the menu? I tried both ways but I still get the same bug. Also, I tried targeting a custom div wrapper ID, but I still get the same result

**Comment by native-apps** [comment]
Was thinking that an extra option to assign the trigger class or ID would do the trick:

``` javascript
// assign the name of the trigger class
    $( '#dl-menu-2' ).dlmenu({
         animationClasses : { classin : 'dl-animate-in-2', classout : 'dl-animate-out-2' },
         backLabel: 'Back',
         useActiveItemAsLink: true,
         triggerClassName: 'dl-trigger-2'
      });      
```

**Comment by nativeimaging** [comment]
Can I paypal you?

**Comment by rejas** [comment]
I could try to add the "multiple menus stuff" into my branch and see what happens :-) Otherwise feel free to send my apul request to my repo if you like.
As for paypaling: you could, but why not send it to some charity instead?
